ci/etc/kokoro/install/project-config.sh

```diff
@@ -27,12 +29,25 @@ declare -A ORIGINAL_COPYRIGHT_YEAR=(
 
 declare -a FROZEN_FILES=()
 
+read -r -d '' QUICKSTART_FRAGMENT < <(
+  for library in $(quickstart_libraries); do
+    tr '\n' '\003' <<_EOF_
+WORKDIR /home/build/quickstart-cmake/${library}
+COPY google/cloud/${library}/quickstart /home/build/quickstart-cmake/${library}
+RUN env -u PKG_CONFIG_PATH cmake -H. -B/i/${library}
+RUN cmake --build /i/${library}
```

Comment:
Do we care that we're losing the `-j ${NCPU:-4}`?

Reply:
I do not think so, there is a single `.cc` in each of these directories, by design.